From 1292724d843f30bf2c2fc578b2cb65654bc4aa21 Mon Sep 17 00:00:00 2001
From: bma <Bo.Ma@windriver.com>
Date: Thu, 30 May 2013 12:03:46 +0800
Subject: Fix invalid assertion for video frames in libstagefright

PFrames spacing could be zero, when encoding video only
using IFrames only. Removing invalid assertion on it.

For:AXIA-2586
Change-Id: Ie1ccfe08ad16e6bd0bc8082d53ac768bf03210d5
Signed-off-by: bma <Bo.Ma@windriver.com>
---
 media/libstagefright/OMXCodec.cpp |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/media/libstagefright/OMXCodec.cpp b/media/libstagefright/OMXCodec.cpp
index fad1d2c..00c1c4f 100644
--- a/media/libstagefright/OMXCodec.cpp
+++ b/media/libstagefright/OMXCodec.cpp
@@ -938,7 +938,6 @@ static OMX_U32 setPFramesSpacing(int32_t iFramesInterval, int32_t frameRate) {
         return 0;
     }
     OMX_U32 ret = frameRate * iFramesInterval - 1;
-    CHECK(ret > 1);
     return ret;
 }
 
-- 
1.7.4.1

